hack/runtime-migrator/README-rr.md

```diff
@@ -1 +1,178 @@
-# Runtime Restorer
\ No newline at end of file
+# Runtime Restore
+
+The `runtime-restore` application has the following tasks:
+1. Connect to a Gardener project, and a KCP cluster.
+2. Retrieve all existing shoot specifications.
+3. For each runtime on the input list:
+   1. Gets the Shoot, Cluster Role Bindings and OpenIDConnect resources from the backup.
+   2.  To prevent KIM from modifying the shoot and the runtime resources, update the Runtime CR to label the runtime as controlled by Provisioner.
+   3. Patch the shoot with the Shoot spec read from the backup.
+   4. Apply the ClusterRoleBindings from the backup if the objects don't exist on the runtime.
+   5. Apply OpenIDConnect from the backup if the objects don't exist on the runtime.
+
+## Build
+
+To build the `runtime-restore` application, run:
+
+```bash
+go build -o ./bin/runtime-restore ./cmd/restore
+``` 
+
+## Usage
+
+### Dry Run
```

Comment:
consider switching the order of code block and explainer text like you did in https://github.com/kyma-project/infrastructure-manager/pull/596/files#r1916205022

Reply:
done